### DISCUSSION TITLE
SLEAP docker

### DISCUSSION THREAD

**Discussion by chiyu1203** [talmolab/sleap#1312] [Ideas]
Hello SLEAP team
I would like to use Sleap to do multi-animal pose estimation (insects). However our department HPC cluster requires us to execute our software via docker support. I wonder is there any plan to release Sleap docker image in the future, or is there any suggestion about packaging the Sleap in docker image?

Best Regards
Chiyu

**Comment by talmo** [comment]
Hi @chiyu1203,

Yep, we use Docker on our cluster as well. As long as you can install stuff in a general purpose container you should be good to go with following the [standard SLEAP installation instructions](https://sleap.ai/installation.html). If you check what your lab or HPC group recommends for running TensorFlow/Python 3.7-3.9, you will be able to just `pip install sleap` in that container.

That said, it'd be a great idea to provide a SLEAP-specific container to make it easier for use cases like these. Thanks for the suggestion!

If you get something working that you like, or if you have any further questions feel free to let us know!

Cheers,

Talmo
